### PR TITLE
SC-9556: Support PHP8.1 compatibility

### PR DESCRIPTION
**Pull request by zyuzka**
Ticket: https://spryker.atlassian.net/browse/SC-9556
RG: https://release.spryker.com/release-groups/view/4298

## Changelog

- introduce support for PHP 8.1
- updated all PHP versions to the latest patch
- updated third-party packages(Tideways, Newrelic)
- removed alpine 3.12 support
- removed PHP 7.3 support
- added support alpine 3.16
- updated dependency order into Dockerfiles
- updated CI
- updated ReadMe

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: debian/bullseye/7.3/Dockerfile [hunk 1]
```diff
@@ -4,61 +4,59 @@ ARG SPRYKER_PHP_VERSION=7.3.33
 FROM php:${SPRYKER_PHP_VERSION}-fpm-bullseye
 
 ARG TARGETPLATFORM
-ARG COMPOSER_VERSION=2.3.4
```
Comment (alexanderM91) [inline debian/bullseye/7.3/Dockerfile:7]: Do we build PHP 7.3?

File: debian/buster/7.3/Dockerfile [hunk 2]
```diff
@@ -4,61 +4,59 @@ ARG SPRYKER_PHP_VERSION=7.3.33
 FROM php:${SPRYKER_PHP_VERSION}-fpm-buster
 
 ARG TARGETPLATFORM
-ARG COMPOSER_VERSION=2.3.4
+ARG COMPOSER_VERSION=2.3.10
```
Comment (alexanderM91) [inline debian/buster/7.3/Dockerfile:7]: The same here?

File: debian/bullseye/7.3/Dockerfile (hunk 1 above)
Comment (zyuzka) [reply debian/bullseye/7.3/Dockerfile:7]: for both Debian - yes

File: debian/buster/7.3/Dockerfile (hunk 2 above)
Comment (zyuzka) [reply debian/buster/7.3/Dockerfile:7]: +

**Review by alexanderM91 — APPROVED** [review]

**Pull request merged**